Buildings/Fluid/CHPs/BaseClasses/AssertFuelFlow.mo

```diff
@@ -0,0 +1,94 @@
+within Buildings.Fluid.CHPs.BaseClasses;
+model AssertFuelFlow "Assert if fuel flow is outside boundaries"
+  extends Modelica.Blocks.Icons.Block;
```

Comment:
Better "Assert whether fuel flow is within boundary" because an assertion checks whether something is true, not whether it is false.